login-buttons/src/main/java/com/shaishavgandhi/loginbuttons/GoogleButton.java

```diff
@@ -0,0 +1,54 @@
+package com.shaishavgandhi.loginbuttons;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Typeface;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+
+public final class GoogleButton extends BaseButton {
+    private final Paint linePaint = new Paint();
+
+    public GoogleButton(Context context, AttributeSet attrs) {
+        super(context, attrs, android.R.color.white, R.drawable.google_logo);
```

Comment:
Will have to use the new Google logo. Doesn't make sense to continue using the Google Plus button